runtime/query.go

```diff
@@ -221,23 +221,6 @@ func populateField(f reflect.Value, value string, props *proto.Properties) error
 			f.Field(0).SetInt(int64(t.Unix()))
 			f.Field(1).SetInt(int64(t.Nanosecond()))
 			return nil
-		case "Duration":
-			if value == "null" {
-				f.Field(0).SetInt(0)
-				f.Field(1).SetInt(0)
-				return nil
-			}
-			d, err := time.ParseDuration(value)
-			if err != nil {
-				return fmt.Errorf("bad Duration: %v", err)
-			}
-
-			ns := d.Nanoseconds()
-			s := ns / 1e9
-			ns %= 1e9
-			f.Field(0).SetInt(s)
-			f.Field(1).SetInt(ns)
-			return nil
```

Comment:
OK I'm starting to think there's something weird going on with my diff viewer. Do you need to rebase your commits on a recent master?

Reply:
these weird unrelated diffs concern me as well. I just recently merged `master` into this feature branch, so I'm not sure why these diffs show up. I'll try to figure this out.

Reply:
Instead of merging, you should be rebasing :-)